examples/mplot3d/box3d.py

```diff
@@ -0,0 +1,78 @@
+"""
+===================
+3D box surface plot
+===================
+
+Given data on a gridded volume ``X``, ``Y``, ``Z``, this example plots the
+data values on the volume surfaces.
+
+The strategy is to select the data from each surface and plot
+contours separately using `.axes3d.Axes3D.contourf` with appropriate
+parameters *zdir* and *offset*.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Define dimensions
+Nx, Ny, Nz = 100, 300, 500
+X, Y, Z = np.meshgrid(np.arange(Nx), np.arange(Ny), -np.arange(Nz))
+
+# Create fake data
+data = (((X+100)**2 + (Y-20)**2 + 2*Z)/1000+1)
+
+kw = {
+    'vmin': data.min(),
+    'vmax': data.max(),
+    'levels': np.linspace(data.min(), data.max(), 10),
+}
+
+# Create a figure with 3D ax
+fig = plt.figure(figsize=(7, 4))
```

Comment:
This is much too wide and leaves a bunch of whitespace on the left; I think it will work out better with
```suggestion
fig = plt.figure(figsize=(5, 4))
```